### DISCUSSION TITLE
Why release with firecreacker binary

### DISCUSSION THREAD

**Discussion by 0x161e-swei** [kata-containers/kata-containers#6403] [Q&A] [answered]
I am curious to learn why kata-container release packages contain firecracker binaries.
Is there any customization of firecracker needed for kata-container to support it?

**Comment by fidencio** [comment, marked as answer]
No, no customization on firecracker, but the Kata Containers version is very much tied to one specific version of Firecracker.
By releasing the binary together, we can ensure the users can simply get our tarball, decompress it, and be able to use Kata Containers with Firecracker in an environment that's prepared for that.
